### PR TITLE
Data defined style preset for composer maps

### LINKED ISSUE #629: [composer] Data defined style preset for composer maps
https://github.com/qgis/QGIS/commit/b87e5f7573752533561f943e2c8333f280aebb69
Author: Nyall Dawson
Date:   Wed Aug 19 20:42:04 2015 +1000
    
    Sponsored by City of Uster

### PR DESCRIPTION
**Pull request by yjacolin**
* Move Data-defined section into general tools

Fix #629

### CODE REVIEW

File: source/docs/user_manual/introduction/general_tools.rst [hunk 1]
```diff
@@ -775,3 +775,34 @@ Removing embedded layers
 ------------------------
 
 Right-click on the embedded layer and choose |removeLayer| :sup:`Remove`.
+
+.. _data-defined:
+
+Data-defined setup
+===================
+
+In some part of the vector layer properties you can find an icon |dataDefined|
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]: this intro is no longer valid

Comment (yjacolin) [inline source/docs/user_manual/introduction/general_tools.rst]: I changed to `In some part of the vector layer properties and composer layout you can find ...`

File: source/docs/user_manual/introduction/general_tools.rst [hunk 2]
```diff
@@ -775,3 +775,34 @@ Removing embedded layers
 ------------------------
 
 Right-click on the embedded layer and choose |removeLayer| :sup:`Remove`.
+
+.. _data-defined:
```
Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]: actually, this reference has been replaced by data_defined (https://github.com/qgis/QGIS-Documentation/pull/1081/commits/78d058d558c409529fad2236211eab8c7a33c794#diff-7780fa8508095c89905755dffa4e1f23R2114)

**Comment by DelazJ** [comment]
@yjacolin could you rebase some of your PRs to ease review, please? Thanks

**Comment by yjacolin** [comment]
Note to myself: add icons yellow and red data defined icons + rebase

**Comment by yjacolin** [comment]
oh no, another amaizing confilcts. Desesperate!

**Comment by DelazJ** [comment]
yellow and red icons of DD are available in #1116.

**Comment by DelazJ** [comment]
merging. Will see for icons later

**Pull request merged**